ClassicLibrary.c

```diff
@@ -16,18 +16,15 @@
 // This swap breaks the max heap property so we "heapify" it again. 
 // We continue until heap size becomes one.
 
-
-/// <summary>Swaps the largest element (root of max heap) with the last element of the unsorted array</summary>
-
+/// <summary>Swaps the largest element with the last element of the unsorted array.</summary>

```

Comment:
This summary is wrong. Your swap method is not fitted for an array, it is a general swap method which would allow to user to swap any two values (which is a good thing). Your comments should indicate that.

Reply:
*for the future